### PR TITLE
Increase minimum asdf version to 2.5.2

### PR DESCRIPTION
**Pull request by eslavich**
### Description

This pull request updates the minimum asdf version to 2.5.2, which adds schemas that were missing from ASDF Standard 1.4.0.  See #9956 for discussion.

### CODE REVIEW

**Comment by nden** [comment]
LGTM. I'll let @bsipocz approve and merge.

File: CHANGES.rst [hunk 1]
```diff
@@ -463,6 +463,7 @@ Other Changes and Additions
 
 - Updated wcslib to v7.1. [#9829]
 
+- Increase minimum asdf version to 2.5.2. [#9996]
```
Comment (bsipocz) [inline CHANGES.rst]: good point, we should have added this entry with the 2.5 change, too, so I suggest to add that other PR number there, too so users can see the reasoning why this change was made into a bugfix release.
```suggestion
- Increase minimum asdf version to 2.5.2. [#9996, #9819]
```

**Comment by bsipocz** [comment]
Thanks @eslavich! I'll add the number for the other PR during the backports!
